packages/synthetic-chain/package.json

```diff
@@ -17,6 +18,7 @@
     "test:xs": "exit 0"
   },
   "dependencies": {
+    "better-sqlite3": "^9.2.2",
```

Comment:
What's the motivation to add this dependency? It's a pretty heavy one to require on each installation of synthetic chain (aka each layer)

Reply:
It's needed for the `vat-status` module that queries the DB to find out about vat states. It's a pretty common use case.

I don't think it will be in every layer because the `.yarn` is mounted as a global cache https://github.com/Agoric/agoric-3-proposals/blob/cb137a7c172a8a5212f3c96d79725e8a6d34cb5f/packages/synthetic-chain/src/cli/dockerfileGen.ts#L108